### DISCUSSION TITLE
How to "clear screen" when using AlaSQL in web browser?

### DISCUSSION THREAD

**Discussion by yasenstar** [agershun/alasql#1918] [Q&A]
I'm using Archi tool which enabled the query with AlaSQL in HTML exported reports, after I run several query statements, the screen is getting full, is there any method that I can clear screen to empty for new query? (like Ctrl+L for some other SQL in command line)

Thanks,
Xiaoqi

**Comment by mathiasrw** [comment]
What is Archi tool?

**Comment by yasenstar** [comment]
Hi Mathias,

Archi is one architecture modeling tool base on ArchiMate language, here [https://github.com/archimatetool/archi/wiki/HTML-export-v3](url) you can find that Archi adopt our AlaSQL into the HTML exported report for querying.

Regards, Xiaoqi

**Comment by mathiasrw** [comment]
You will have to ask the people who made Archi.
